centaur/src/main/resources/standardTestCases/failures.restart_while_failing_tes.test

```diff
@@ -0,0 +1,27 @@
+name: failures.restart_while_failing_tes
+testFormat: WorkflowFailureRestartWithoutRecover
+callMark: restart_while_failing.B1
+backends: [TES]
+tags: [localdockertest]
```

Comment:
yeah I agree.. ultimately I think instead of targeting backends specifically we should categorize by feature. For example "only run this test on backends that support recover", or "support gcs" etc..